Validation/MtdValidation/plugins/MtdTracksValidation.cc

```diff
@@ -287,25 +350,50 @@ void MtdGlobalRecoValidation::bookHistograms(DQMStore::IBooker& ibook,
       ibook.book1D("TrackETLEffPtMtdZneg", "Track efficiency vs pt (Mtd) (-Z);pt_{RECO} [GeV]", 50, 0, 10);
   meETLTrackEffPtMtd_[1] =
       ibook.book1D("TrackETLEffPtMtdZpos", "Track efficiency vs pt (Mtd) (+Z);pt_{RECO} [GeV]", 50, 0, 10);
+  meETLTrackPtRes_ =
+      ibook.book1D("TrackETLPtRes", "Track pT resolution;pT_{Gentrack}-pT_{MTDtrack}/pT_{Gentrack} ", 100, -0.1, 0.1);
+  meTracktmtd_ = ibook.book1D("Tracktmtd", "Track time from TrackExtenderWithMTD;tmtd [ns]", 100, -1.5, 1.5);
+  meTrackSigmatmtd_ =
+      ibook.book1D("TrackSigmatmtd", "Time Error from TrackExtenderWithMTD; #sigma_{tmtd} [ns]", 100, 0, 0.1);
+  meTrackt0Pid_ = ibook.book1D("Trackt0Pid", "Track t0 as stored in TofPid;t0 [ns]", 100, -1, 1);
+  meTrackSigmat0Pid_ =
+      ibook.book1D("TrackSigmat0Pid", "Sigmat0 as stored in TofPid; #sigma_{t0} [ns]", 100, -0.02, 0.06);
```

Comment:
@gsorrentino18 what is the purpose of negative errors? And why 100 bins for 80 ps? Would not be preferable a bin size of 1 ps on a range [0,0.1]? Obviously for all the uncertainties